wger/manager/views/workout.py

```diff
@@ -121,6 +130,165 @@ def view(request, pk):
+                                                    '<br>'.join(information) + '</p><hr>'
+                            table_content += '</td>'
+                            # table_content += '<td><a href="' + reverse('manager:workout:import_workout', kwargs={
+                                                                    #    'imported_from': imported_from}) + '" class="btn btn-primary">Adopt workout</a></td>'
+                    table_content += '</td>'
+        return render(request, 'workout/imports.html', {'imports': imported_workouts, 'table_content': table_content, 'user': request.user, 'form': form, })
+    return render(request, 'workout/imports.html', {'message': 'No imported workouts currently', 'user': request.user, 'form': form, })
+
+
+# @login_required
+# def import_workout(request, imported_from):
+#     '''
+#     Add an imported workout and redirect to its page
+#     '''
+#     workout = Workout()
+#     workout.user = request.user
+#     workout.imported_from = str(imported_from)
+#     workout.save()
+
+#     imported_json_files = ImportJsonDocument.objects.filter(user=request.user)
+#     for imported_json_file in imported_json_files:
+#         with open(os.path.normpath(str(imported_json_file.json_document)), 'r') as json_doc:
+#             data = json_doc.read()
+#             workouts = json.loads(data)
+#             for workout_info in workouts:
+#                 workout_days = Day()
+#                 for key, value in workout_info.items():
+#                     if key == 'workout_description':
+#                         workout_days.description = value
+#                         workout_days.save()
+#                     if key == 'id':
+#                         # need to fix this line #
+#                         workout_set = Workout.objects.filter(
+#                             imported_from=str(imported_from)).first()
+#                         workout_days.training = workout_set
+#                     if key == 'days':
+#                         workout_days_of_week = value.strip("[]").split(',')
+#                         for day_in_json in workout_days_of_week:
+#                             for day_of_week in DaysOfWeek.objects.all():
+#                                 if day_in_json.strip() == day_of_week.day_of_week:
+#                                     workout_days.day.add(day_of_week)
+#                     if key == 'set_info':
+#                         for info in value:
+#                             for v in info.values():
+#                                 for item in v:
+#                                     for a,b in item.items(): 
+                                        
+#                                         print('##############################')
+#                                         print(a,':::::::::::::::',b)
+#                                         print('##############################')   
```

Comment:
Please remove the print statements from these lines.

Reply:
Thanks, I was debugging but it was resolved.